code/game/mecha/mecha.dm

```diff
@@ -841,6 +842,8 @@
 			user << "<span class='warning'>You can't enter the exosuit with other creatures attached to you!</span>"
 		else
 			moved_inside(user)
+			if(user.fov)
+				user.fov.alpha = 0 //snowflake for vision cone

```

Comment:
this is so that people inside mecha aren't shown? this should be handled in ```Move()```, so that it works for all objects people can go inside of.